build.sbt

```diff
@@ -24,11 +24,13 @@ lazy val metaserver = project
     resolvers += "dhpcs at bintray" at "https://dl.bintray.com/dhpcs/maven",
     libraryDependencies ++= List(
       "io.monix" %% "monix" % "2.3.0",
+      "com.lihaoyi" %% "pprint" % "0.5.3",
+      "com.thoughtworks.qdox" % "qdox" % "2.0-M7", // for java ctags
       "io.get-coursier" %% "coursier" % coursier.util.Properties.version,
       "io.get-coursier" %% "coursier-cache" % coursier.util.Properties.version,
       "ch.epfl.scala" % "scalafix-cli" % "0.5.3" cross CrossVersion.full,
-      "com.geirsson" %% "scalafmt-core" % "1.3.0"
+      "org.scalatest" %% "scalatest" % "3.0.3" % "test",
```

Comment:
super minor, but I believe the latest sbt convention is to use `% Test` instead of `% "test"`